src/main/java/kr/codesqaud/cafe/service/user/UserService.java

```diff
@@ -48,13 +51,22 @@ public UserUpdateForm findUpdate(Long id) {
         return UserUpdateForm.form(user);
     }
 
+    private User findUserId(String userId) {
+        return userRepository.findByUserId(userId)
+                .orElseThrow(() -> new NoSuchElementException("이 유저 아이디를 찾을 수 없어: " + userId));
+    }
+
     private User findUser(Long id) {
         // Optional을 스트림으로 처리
         return userRepository.findById(id)
                 .orElseThrow(() -> new NoSuchElementException("이 아이디를 찾을 수 없어: " + id));
     }
 
     public void updateUser(Long id, UserUpdateForm updateUser) {
+        if (!updateUser.getExistingPassword().equals(updateUser.getPassword())) {
```

Comment:
객체지향에서는 클래스가 가지고 있는 데이터를 통한 로직(연산) 들을 수행할때 클래스에게 데이터에 대한 책임을 주는 것이 보통 기본인데요. 그래서 이런경우도 아래와 같이 캡슐화를 하는것이 좋습니다.

```suggestion
        if (!updateUser.isSamePassword(updateUser.getPassword())) {
```

Reply:
확실히 제가 한 것처럼 하면 말씀해주신 이유와 더불어 가독성도 좋지 않은 것 같습니다! 바로 수정했습니다 :)

- [수정 커밋](https://github.com/won4885/be-java-cafe-max/commit/6a37c238faf194822d3aea1d88aee89b9b42eef2)